sklearn/covariance/graph_lasso_.py

```diff
@@ -204,7 +204,7 @@ def graphical_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
         # https://github.com/scikit-learn/scikit-learn/issues/4134
         d_gap = np.inf
         # set a sub_covariance buffer
-        sub_covariance = np.ascontiguousarray(covariance_[1:, 1:])
+        sub_covariance = np.ascontiguousarray(covariance_[1:, 1:]).copy()
```

Comment:
`np.copy(_, order='C')` ?

Reply:
more concise and guaranteed to copy only once, while this version might copy twice, right?

Reply:
indeed, thanks!